src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -233,17 +247,190 @@ function RemoteFunctions(experimental) {
             }
             return false;
         },
-
         _makeHighlightDiv: function (element, doAnimation) {
             var elementBounds = element.getBoundingClientRect(),
                 highlight = window.document.createElement("div"),
                 styles = window.getComputedStyle(element);
+		var transitionDuration = parseFloat(styles.getPropertyValue('transition-duration'));
```

Comment:
Nit: these two variables could also be combined with the above by separating the values with , like the others are.